Library/Formula/cgdb.rb

```diff
@@ -18,9 +18,14 @@ class Cgdb < Formula
     depends_on "automake" => :build
   end
 
+  if OS.linux?
+    depends_on "flex" => :build
+  end
   depends_on "help2man" => :build
   depends_on "readline"
 
+  patch :DATA
```

Comment:
Fixes specifically for Linux that don't affect Mac should be submitted to Linuxbrew. I should have bene more specific. I meant, have you submitted this patch upstream to the maintainers of cgdb?